shared/src/main/scala/squants/motion/Pressure.scala

```diff
@@ -68,18 +70,32 @@ object StandardAtmospheres extends PressureUnit {
   val conversionFactor = Newtons.conversionFactor * 1.01325e5
 }
 
+object MillimeterOfMercury extends PressureUnit {
+  val symbol = "mmHg"
+  val conversionFactor = Newtons.conversionFactor * 133.322387415
+}
+
+object Torr extends PressureUnit {
+  val symbol = "Torr"
+  val conversionFactor = Newtons.conversionFactor * 101325d / 760d
```

Comment:
A Torr is "defined as exactly 1 / 760 of a standard atmosphere" (per wikipedia). So I wonder if `conversionFactor` would be better defined as `StandardAtmospheres.conversionFactor / 760d` ? It won't matter for correctness, but might make the definition easier to read.